### ISSUE TITLE
Zoom-to-Feature Functionality with Shortcut

### ISSUE THREAD

**Issue by MisterGC** [ndsev/erdblick#7]
The map viewer component should be enhanced to incorporate a 'zoom-to-feature' function. This feature allows users to quickly focus on and inspect specific 2D and 3D represented map features in more detail.

Users should be able to select a specific map feature or features, either by directly clicking on them or by entering their feature IDs, and then execute the 'zoom-to-feature' command. The map viewer should then automatically adjust the view to focus on and zoom into the selected features.

To enhance usability and efficiency, this feature should be coupled with a shortcut key or key combination, which can be used to quickly activate the 'zoom-to-feature' command. In addition a lock-to-feature camera could help to easily orbit arround the feature and zoom-in/out for a detailed in vestigation.

This 'zoom-to-feature' functionality will enable users to more quickly and easily inspect individual or grouped map features in detail, improving the overall map exploration and interaction experience.

**Comment by josephbirkner** [comment]
I would rephrase this as "Zoom to Selection"
